### DISCUSSION TITLE
Druid validator(Obsrv BB) installation failed | Ed Easy Installer

### DISCUSSION THREAD

**Discussion by Marc70008** [Sunbird-Ed/Community#539] [Installation] [answered]
Hi, I am following the Ed Easy Installer script to provision Ed platform
While running the Ed easy installer bash script, druid-validator(Obsrv BB) installation failed due to mount volume error as the **flink-conf** key is missing in the configmap
what should be done here?

**Comment by Krishnaj20** [comment]
@gandham-santhosh - for your attention please

**Reply by tsprasath** [reply, marked as answer]
@Marc70008 Druid validator is not required for now. It will be used in the next phase. Ignore it for now.

**Reply by Marc70008** [reply]
got it @tsprasath

**Reply by tsprasath** [reply]
@Marc70008  Let me know if any query you have if not please close the thread and mark it as answer
